ocs/constants.py

```diff
@@ -36,3 +36,4 @@
 
 # Other
 SECRET = "Secret"
+ADMIN_BASE64 = 'YWRtaW4='
```

Comment:
Shouldn't this be generated automatically? Should everybody have the same key?

Reply:
> Shouldn't this be generated automatically? 
No need of generating automatically everytime value is same
> Should everybody have the same key?
Yes everybody can use this in cephfs storageclass creation

Reply:
@prsurve This doesn't seem to match the way customer are going to use the product.

Reply:
Hi @mbukatov we use ADMIN_BASE64 here [0]  if we want we can create user at ceph side and can assign to [1] but  we need to provide rule like which pool can be access by that user by default admin has access to all pool

[0] https://github.com/red-hat-storage/ocs-ci/blob/master/templates/CSI/cephfs/secret.yaml#L13
[1] https://github.com/red-hat-storage/ocs-ci/blob/master/templates/CSI/cephfs/secret.yaml#L9